### PR TITLE
Technical/aa 1796 add configuration provider to updater

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: HappyTravel.Nakijin.Updater/appsettings.json [hunk 1]
```diff
@@ -67,6 +67,9 @@
       "DbCommandTimeOut": "CS_DB_COMMAND_TIME_OUT"
     }
   },
+  "SupplierOptionsProvider": {
+    "Endpoint": "CS_SUPPLIER_OPTIONS_SERVICE_ENDPOINT"
```
Comment (hasmikharutyunyan) [inline HappyTravel.Nakijin.Updater/appsettings.json:71]: Is it permanent solution ?

Comment (mingazhev) [reply HappyTravel.Nakijin.Updater/appsettings.json:71]: Not sure. We discussed about using consul in updaters, but it is not added to this updater yet.
I think all the things we set from env vars should be moved to consul later

Comment (hasmikharutyunyan) [reply HappyTravel.Nakijin.Updater/appsettings.json:71]: I think it is better to use vault, that kind of data we use from vault now

Comment (mingazhev) [reply HappyTravel.Nakijin.Updater/appsettings.json]: Fixed

**Review by hasmikharutyunyan — APPROVED** [review]

**Pull request merged**